src/tribler-core/tribler_core/components/version_check.py

```diff
@@ -17,5 +19,5 @@ async def run(self):
         self.version_check_manager.start()
 
     async def shutdown(self):
```

Comment:
**nit**: it doesn't relate to the PR, but it seems that `await super().shutdown()` is missed